### ISSUE TITLE
Document the contribution process

### ISSUE THREAD

**Issue by dpursehouse** [GerritCodeReview/docker-gerrit#4]
Having the project mirrored here on Github can lead people to incorrectly assume that code reviews can be submitted by PR, when actually we expect them on gerrit-review.googlesource.com.

**Comment by lucamilanesio** [comment]
Yes, shall we do it for every Gerrit Code Review project though? This problem happened with other repos as well :-(

**Comment by dpursehouse** [comment]
That's a good point.  Having said that, it's pretty much standard for each open source project to have its own contribution process document isn't it?

**Comment by lucamilanesio** [comment]
Oh yes, let me dig into what we have already in place ;-)

**Comment by davido** [comment]
If all other option fail, just say in the documentation: readonly, upload your changes to this gerrit instance, like it is done here: https://github.com/davido/gerrit-oauth-provider.

**Comment by dpursehouse** [comment]
Any update on this?  It's been open for over a year.

**Comment by dpursehouse** [comment]
Never mind, I see a commit was added in 1f60cf8febb6b62f43f16ae9459e08c51ba1a5b2

It doesn't seem to have stopped people from submitting issues though :/

Can we just disable issues on this project?

**Comment by davido** [comment]
> Can we just disable issues on this project?

I think this is possible.
